### ISSUE TITLE
Error in calculating generator loss while training hifigan vocoder for libriTTS

### ISSUE THREAD

**Issue by coding-phoenix-12** [speechbrain/speechbrain#2241]
### Describe the bug

The error I got was 
 File "speechbrain/speechbrain/lobes/models/HifiGAN.py", line 1469, in forward
    print(y_hat.shape, y.shape)
AttributeError: 'list' object has no attribute 'shape'

On further investigation, I found in file speechbrain/recipes/LibriTTS/vocoder/hifigan/train.py line no 65
```
loss_g = self.hparams.generator_loss(
             y_hat, y, scores_fake, feats_fake, feats_real
        )
```

but the generator loss function in file speechbrain/speechbrain/lobes/models/HifiGAN.py line no 1438

```
def forward(
        self,
        stage,
        y_hat=None,
        y=None,
        scores_fake=None,
        feats_fake=None,
        feats_real=None,
        log_dur_pred=None,
        log_dur=None,
    ):
```
takes stage as a parameter

### Expected behaviour

Changing file speechbrain/recipes/LibriTTS/vocoder/hifigan/train.py line no 65 to 
```
loss_g = self.hparams.generator_loss(
             stage, y_hat, y, scores_fake, feats_fake, feats_real
        )
```
will fix it.

**Comment by BenoitWang** [comment]
Hi @coding-phoenix-12 thanks for the report. In fact I already spotted this problem during my review of the pull request [here](https://github.com/speechbrain/speechbrain/pull/2044#discussion_r1281975272), could you please do a fast fix @Chaanks? Thanks!

**Comment by BenoitWang** [comment]
Fixed in #2244 , thank you @coding-phoenix-12 @Chaanks !
